icevision/models/mmdet/lightning/model_adapter.py

```diff
@@ -59,16 +66,16 @@ def validation_step(self, batch, batch_idx):
                 imgs=[data["img"]], img_metas=[data["img_metas"]]
             )
 
-        preds = self.convert_raw_predictions(
-            batch=data, raw_preds=raw_preds, records=records
+        preds = convert_raw_predictions(
+            batch=data,
+            raw_preds=raw_preds,
+            records=records,
+            detection_threshold=self.detection_threshold,
         )
         self.accumulate_metrics(preds)
 
         for k, v in outputs["log_vars"].items():
             self.log(f"valid/{k}", v)
 
-        # TODO: is train and eval model automatically set by lighnting?
-        self.model.train()
```

Comment:
Why did you remove this line?

Reply:
PyTorch Lightning automatically takes care of `model.train()` and `model.eval()`. I removed it to make it consistent with the other lightning adapters

Reply:
@potipot do you recall why this was there in the first place?
I think @Anjum48 point is valid.